packages/avs/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "avs"
```

Comment:
There are lots of avs packages.. let's make some more globally unique name.

`lavs-wasi`, `lwasi-std`, `layer-wasi-awesomeness`. 
I am not too picky about naming, but let's make them something we can claim on crates.io and unique in all our repos